frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx

```diff
@@ -339,6 +339,7 @@ const getK8sWatchResources = (
 
 export const ProvidedAPIsPage = (props: ProvidedAPIsPageProps) => {
   const { t } = useTranslation();
+  const match = useRouteMatch();
```

Comment:
should we use here the `useParams` hook as well?

Reply:
`useParams` doesn't work for our needs here. I later use `match.url` to build a link. `useParams` gives us the `match.params` property.